docs/development.md

```diff
@@ -7,7 +7,7 @@
 Install required tools:
```

Comment:
The help text just above here is confusing. I'm not quite sure what to do w/
`- Install cmake or (optionally, required tools for GPUs)`

is the `or` for the other tools for GPUs, or for `go generate/go build`?

Reply:
I'll tidy this part of the README up.